ui/app/AppLayouts/Profile/views/wallet/ManageTokensView.qml

```diff
@@ -128,10 +133,74 @@ ColumnLayout {
     }
 
     Component {
-        id: supportedTokensListPanel
-        SupportedTokenListsPanel {
-            sourcesOfTokensModel: root.sourcesOfTokensModel
-            tokensListModel: root.tokensListModel
+        id: advancedTab
+        ColumnLayout {
+            spacing: 0
+            StatusBaseText {
+                Layout.fillWidth: true
+                Layout.topMargin: 18
+                Layout.bottomMargin: 18
+                text: qsTr("Token lists")
+                color: Theme.palette.baseColor1
+            }
+            SupportedTokenListsPanel {
+                Layout.fillWidth: true
+                Layout.fillHeight: true
+                sourcesOfTokensModel: root.sourcesOfTokensModel
+                tokensListModel: root.tokensListModel
+            }
+            StatusBaseText {
+                Layout.fillWidth: true
+                Layout.topMargin: 40 + 18
+                Layout.bottomMargin: 26
+                text: qsTr("Asset settings")
+                color: Theme.palette.baseColor1
+            }
+            StatusDialogDivider {
+                Layout.fillWidth: true
+            }
+            StatusListItem {
```

Comment:
From the design, it's not clear to me that these 2 switches options should be displayed as selectable list elements. At least, probably the background should not be changed when hovering. I mean that:
<img width="779" alt="Screenshot 2024-01-11 at 10 18 15" src="https://github.com/status-im/status-desktop/assets/97019400/a0cfd38e-b528-480f-ae6a-656839e5fb3b">

Reply:
Yeah this is kinda unclear from the design what the color should be when hovering; it's unfortunate that the input bg color is the same as the list item's hover color in our palette :/ Something for @benjthayer to decide?

Reply:
Also, as they are standard `StatusListItem`s they ought to default to have the hover enabled (just like anywhere else in the Settings)